doc/Type/Int.pod6

```diff
@@ -81,6 +81,12 @@ can accept non-C<Int> arguments, which will be coerced to C<Int>.
     say expmod(4, 2, 5);    # OUTPUT: «1␤»
     say 7.expmod(2, 5);     # OUTPUT: «4␤»
 
+C<$y> argument can also be negative, in wich case, the result is
+equivalent to C< ($x ** $y) mod $mod.
```

Comment:
Closing > is missing. Please use Perl6-mode for Atom that will warn you about these things.